Makefile

```diff
@@ -912,6 +912,10 @@ test: $(SETUP_ENVTEST) ## Run unit and integration tests with race detector
 	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test -race ./... $(TEST_ARGS)
 	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test -run "^TestFuzzyConversion$$" ./... $(TEST_ARGS)
 
+.PHONY: test-conversions
+test-conversions: $(SETUP_ENVTEST) ## Run conversions test
+	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test -run "^TestFuzzyConversion$$" ./... $(TEST_ARGS)
```

Comment:
nit, now that we have a separate make target for test-conversion, lets call it in make test (instead of duplicating the command in L913)